Ory/kratosauth.yml

```diff
@@ -38,10 +38,10 @@ services:
   kratos-selfservice-ui-node:
     image: oryd/kratos-selfservice-ui-node:latest
     environment:
-      - KRATOS_PUBLIC_URL=http://kratos:4433/
-      - KRATOS_BROWSER_URL=http://127.0.0.1:4433/
+      - KRATOS_PUBLIC_URL=http://kratos.openfoodfacts.localhost:4433/
+      - KRATOS_BROWSER_URL=http://kratos.openfoodfacts.localhost:4433/
     ports:
-      - '127.0.0.1:4455:3000'
+      - '4455:3000'
```

Comment:
Please keep as it was here (this line), it's not linked and it's better for security to keep exposed only on localhost.